### PR TITLE
create header and add favicon

### LINKED ISSUE #17: feat/ui-create-header
**AC**
Create <Header/> component according to designs (https://www.figma.com/file/RkzObeqWU4mYZwuYVnnPOC/TeamUp?node-id=52412%3A16288)

Include header in the base component

### PR DESCRIPTION
**Pull request by Mariusz-web**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -1,43 +1,134 @@
...
+            <AppBar position='static'>
+                <Toolbar>
+                    <Typography
+                        variant='h5'
+                        component='div'
+                        sx={{ flexGrow: 1 }}
+                    >
+                        TeamUp
+                    </Typography>
+                    <div>
```
Comment (Veryyapeee) [inline src/components/Header/Header.tsx]: We don't do div in here 8)

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -1,43 +1,134 @@
...
+                                    variant='body1'
+                                    color={'#000'}
+                                    component={NavLink}
+                                    to={ROUTES.HOME}
+                                >
+                                    Home
+                                </Link>
+                            </MenuItem>
+                            <MenuItem onClick={handleClose}>
+                                <Link
```
Comment (Veryyapeee) [inline src/components/Header/Header.tsx]: styles and logic kinda repeats, extract that to separate component (Atom)

File: src/components/Header/Header.tsx [hunk 3]
```diff
@@ -1,43 +1,134 @@
...
+import Typography from '@mui/material/Typography';
+import * as React from 'react';
 import { NavLink } from 'react-router-dom';
 
 import { ROUTES } from '../../routes/Routes';
 
 export const Header = () => {
     const id = 1;
+
+    const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
```
Comment (Veryyapeee) [inline src/components/Header/Header.tsx]: just import useState from react

File: src/components/Header/Header.tsx [hunk 4]
```diff
@@ -1,43 +1,134 @@
+import MenuIcon from '@mui/icons-material/Menu';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import IconButton from '@mui/material/IconButton';
+import Link from '@mui/material/Link';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import * as React from 'react';
```
Comment (Veryyapeee) [inline src/components/Header/Header.tsx]: since react 16.x we don't need to import * from react

File: src/components/Header/Header.tsx [hunk 5]
```diff
@@ -1,43 +1,134 @@
...
 import { NavLink } from 'react-router-dom';
 
 import { ROUTES } from '../../routes/Routes';
 
 export const Header = () => {
     const id = 1;
+
+    const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+    const open = Boolean(anchorEl);
```
Comment (Veryyapeee) [inline src/components/Header/Header.tsx]: what do you need this for?
You don't need that reassign in here

File: src/components/Header/Header.tsx [hunk 6]
```diff
@@ -1,43 +1,134 @@
...
+                    </Typography>
+                    <div>
+                        <IconButton
+                            size='large'
+                            edge='start'
+                            color='inherit'
+                            aria-label='menu'
+                            sx={{ mr: 2, background: 'rgba(0,0,0,0.20)' }}
+                            id='basic-button'
+                            aria-controls={open ? 'basic-menu' : undefined}
```
Comment (Veryyapeee) [inline src/components/Header/Header.tsx]: Couldn't you just
`open && 'basic-menu'`

File: src/screens/Home/Home.tsx [hunk 7]
```diff
@@ -1,11 +1,9 @@
-import { NavLink } from 'react-router-dom';
-
-import { ROUTES } from '../../routes/Routes';
+import { Header } from '../../components';
 
 export const Home = () => {
     return (
         <>
-            <NavLink to={ROUTES.FEED}>FEED</NavLink>
+            <Header />
```
Comment (Veryyapeee) [inline src/screens/Home/Home.tsx]: why is it here? It's already included in <Base/> where it belongs

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/components/Atoms/Navigation.tsx [hunk 8]
```diff
@@ -0,0 +1,64 @@
+import { MenuItem } from '@mui/material';
+import Link from '@mui/material/Link';
+import { NavLink } from 'react-router-dom';
+
+import { ROUTES } from '../../routes/Routes';
+
+type NAV_ITEM = {
```
Comment (Veryyapeee) [inline src/components/Atoms/Navigation.tsx]: Type should start just with capital letter, what you did here is convention for constants

File: src/components/Atoms/Navigation.tsx [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+    src: string;
+    label: string;
+    to: string;
+};
+
+const NAV_ITEMS = [
```
Comment (Veryyapeee) [inline src/components/Atoms/Navigation.tsx:13]: Just
navItems

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

**Comment by Veryyapeee** [comment]
https://user-images.githubusercontent.com/43120132/162260805-ec3533ec-9b4a-4978-a579-9c5faff0e95f.MOV

Looks kinda buggy on preview

File: src/components/Atoms/Navigation.tsx (hunk 9 above)
Comment (Veryyapeee) [reply src/components/Atoms/Navigation.tsx:13]: You didn't changed that line
`navItems`

File: src/components/Atoms/Navigation.tsx [hunk 10]
```diff
@@ -0,0 +1,64 @@
+import { MenuItem } from '@mui/material';
+import Link from '@mui/material/Link';
+import { NavLink } from 'react-router-dom';
+
+import { ROUTES } from '../../routes/Routes';
+
+type navItem = {
```
Comment (Veryyapeee) [inline src/components/Atoms/Navigation.tsx]: `NavItem`

**Review by Veryyapeee — APPROVED** [review]
Approved, looks ok I think, please just fix these two small comments

**Review by T0ffik — APPROVED** [review]

**Pull request merged**